radix-engine-interface/src/model/resource/non_fungible_local_id.rs

```diff
@@ -172,9 +188,9 @@ impl<E: Encoder<ScryptoCustomValueKind>> Encode<ScryptoCustomValueKind, E> for N
                 encoder.write_size(v.len())?;
                 encoder.write_slice(v.as_bytes())?;
             }
-            NonFungibleLocalId::Number(v) => {
+            NonFungibleLocalId::Integer(v) => {
                 encoder.write_byte(1)?;
-                encoder.write_slice(&v.to_le_bytes())?;
+                encoder.write_slice(&v.to_be_bytes())?; // TODO: variable length encoding?
```

Comment:
Out of interest, why are we using `be_bytes` here but `le_bytes` for all other numeric types in sbor?

Reply:
No strong reason, but to make the endian-ness (within non-fungible id) consistent.